CMakeLists.txt

```diff
@@ -205,6 +207,98 @@ if (NN_ENABLE_NANOCAT)
     target_link_libraries (nanocat ${PROJECT_NAME})
 endif ()
 
+find_program (ASCIIDOCTOR_EXE asciidoctor)
```

Comment:
Should this be inside the conditional block starting at 221? In other words, don't even check if it's installed unless doc build is enabled. Then, if this is moved inside that block, a fatal error could be issued, rather than just a `WARNING`